pkg/shell/shell.go

```diff
@@ -238,6 +240,14 @@ func WithBundleFile(bundleFilename string, targetFilename string) OptionFunc {
 	}
 }
 
+// WithBundleLabel sets the label and optional tags for bundle operations
+func WithBundleLabel(label string, tags ...string) OptionFunc {
+	return func(o *Options) {
+		o.gatecheck.label = label
+		o.gatecheck.tags = tags
+	}
+}
+
```

Comment:
I'm not opposed to labeling and tagging artifacts added to the gatecheck bundle (I don't like using filenames alone for inferring semantic meaning), however I don't see this being used anywhere.